src/main/resources/db/migration/V8__add_weather_trigger.sql

```diff
@@ -0,0 +1,16 @@
+CREATE OR REPLACE FUNCTION check_weather()
```

Comment:
I think it should be implemented on code level when you import data into database. Stored procedures/triggers etc are non-obvious and usually not encouraged.

Reply:
Removed. Importing weather data is Alexandr part, so, i believe, he will add it later in subproject

Reply:
Ok. As "Action item", could you please create ticket/issue whatever to be sure that this task will be implemented and nothing will be forgotten and post link in reply to this comment. Thanks

Reply:
https://app.vivifyscrum.com/boards/61175/NEAF-74